src/App.js

```diff
@@ -1,24 +1,29 @@
-import React from 'react'
-import { BrowserRouter, Route } from 'react-router-dom'
+import React, { Suspense } from 'react'
+import { BrowserRouter as Router, Route, Switch } from 'react-router-dom'
 
-import './App.scss'
+import { PATH_URL } from 'common/constants'
 import ImgGallery from 'common/components/img-gallery/'
```

Comment:
for this as well, I would expect to have this kind of import:

```
import { ImageGallery } from 'common/components'
```

I think it's better to call it **ImageGallery** instead of **ImgGalerry**, what do you think ?